neural_nets/backprop.py

```diff
@@ -133,7 +139,7 @@ def training_step(self, inputs, desired_outputs):
         plt.figure()
         # 10 separate runs
         for _ in range(10):
-            nn = BackPropNet([3, 5, 2], activation="tanh")
+            nn = BackPropNet([3, 5, 2], activation=["tanh","tanh"])
```

Comment:
In theory, saying `activation="tanh"` which is what it was before, and saying `activation=["tanh","tanh"]` should mean the same.. Is there a reason you tried this?

Reply:
Yes, I know it's the same.  I did not mean to change it.  I merely left it like that after I tried other combinations, e.g., ["relu", "tanh"].